internal/http/middleware.go

```diff
@@ -0,0 +1,42 @@
+package http
```

Comment:
Usually we don't use http directory in Go. It's better to create separate directories inside internal: handlers, middleware, models